chart/values.yaml

```diff
@@ -675,6 +675,8 @@ nats:
   cluster:
     enabled: true
     replicas: 3
+  # Cluster Domain configured on the kubelets
+  k8sClusterDomain: cluster.local
```

Comment:
wondering if there's other places which would this need fixed, and if so can we have a global variable for this? Although not sure if that's possible in helm IIRC?

Reply:
This is not guaranteed to be `cluster.local` for all clusters.

Reply:
Yes it is not guaranteed to be cluster.local for all clusters. But can we add to for the user to know where to change the domain in Nats config? 
Where can we add the global variable?

Reply:
@tiagolobocastro Where can we add the global variable?

Reply:
I'm not sure, we could add this to like the base config, but then we can't use it here right? Because we can't reference variables here IIRC?

Reply:
Yes, it's been difficult to set values to the values.yaml using variables. May be not possible at the moment.

Reply:
I'm not sure how to go about doing this. I'm going to say I'm good with `custer.local` at the moment.

Reply:
Alright, I'm good with that.
I also want to check, what is the intent in adding this in the yaml, just as documentation? If so the comment should start with # --

Reply:
I removed k8sClusterDomain and set 'useFQDN' to 'false' now.